equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/model/report/Crm5UpperLimitReportModel.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.justice.laa.crime.equinity.historicaldata.model.report;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.Immutable;
+
+import java.time.LocalDate;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Immutable
+@Entity(name="Crm5UpperLimitReport")
```

Comment:
Just a suggestion
possible to put all `Entity` name in a Const class or interface to re-use it?
`@Entity(name=CRM5_UPPER_LIMIT_REPORT)`

Reply:
You can't use variables on annotations.  Annotations only accept fixed values